{
  try {
    testRWRebalanceWithReplication(false);
  }
 catch (  AssertionError ae) {
    logger.error("Assertion broken in testRWRebalanceWithReplication ",ae);
    throw ae;
  }
}
